internal/service/account/resource_account_authentication_test.go

```diff
@@ -35,6 +46,109 @@ func TestAccAivenAccountAuthentication_basic(t *testing.T) {
 	})
 }
 
+func testAccAccountAuthenticationResourceSAMLCertificate(rName, certificate string) string {
+	return fmt.Sprintf(`
+resource "aiven_account" "user" {
+  name = "test-acc-account-%[1]s"
+}
+
+resource "aiven_account_authentication" "method" {
+  account_id       = aiven_account.user.account_id
+  type             = "saml"
+  name             = "test-acc-auth-method-%[1]s"
+  saml_certificate = <<-EOT
+  %[2]s
+  EOT
+}
+`, rName, certificate)
+}
+
+func TestAccAivenAccountAuthentication_saml_valid_certificate(t *testing.T) {
```

Comment:
I still this it is a good idea to combine `TestAccAivenAccountAuthentication_saml_valid_certificate` and `TestAccAivenAccountAuthentication_saml_valid_certificate_update` since `TestAccAivenAccountAuthentication_saml_valid_certificate` is a first step for the `TestAccAivenAccountAuthentication_saml_valid_certificate_update`